src/visor/unconfirmed.go

```diff
@@ -339,28 +338,30 @@ func (utp *UnconfirmedTxnPool) InjectTxn(bc *Blockchain, t coin.Transaction) (bo
 	known := false
 	utp.txns.update(h, func(tx *UnconfirmedTxn) {
 		known = true
-
 		now := utc.Now().UnixNano()
 		tx.Received = now
 		tx.Checked = now
-
-		if unspentNotExistErr == nil {
-			tx.IsValid = 1
-		} else {
-			tx.IsValid = 0
-		}
+		tx.IsValid = 1
```

Comment:
We don't save unconfirmedtxns missing unspents now?

Reply:
1. Yes, we don't save the tx that missing unspents now, actually we will check unconfirmed txs and remove those missing unspents txns when node start. 
2. VerifyTransactionFee method won't return ErrUnspentNotExist, and no one returns `ErrUnspentNotExist`.